ci/properties/terraform.properties.json

```
@@ -1,6 +1,7 @@
 {
     "name": "Terraform",
     "description": "Set up Terraform CLI in your GitHub Actions workflow.",
+    "creator": "Hashicorp",
```

```suggestion
    "creator": "HashiCorp",
```
see: https://www.hashicorp.com/